postgres-driver/listener_test.go

```diff
@@ -0,0 +1,137 @@
+package postgresdriver
+
+import (
+	"testing"
+
+	"github.com/pokt-foundation/portal-api-go/repository"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPostgresDriver_ListenApplication(t *testing.T) {
+	c := require.New(t)
+
+	listenerMock := NewListenerMock()
+	driver := NewPostgresDriverFromSQLDBInstance(nil, listenerMock)
+
+	listenerMock.MockEvent(ActionInsert, ActionUpdate, &repository.Application{
+		ID: "321",
+		GatewayAAT: repository.GatewayAAT{
+			Address: "123",
+		},
+		GatewaySettings: repository.GatewaySettings{
+			SecretKey: "123",
+		},
+		NotificationSettings: repository.NotificationSettings{
+			Full: true,
+		},
+	})
+
+	n := <-driver.Notification
+	c.Equal(ActionInsert, n.Action)
+	c.Equal(TableApplications, n.Table)
+	app := n.Data.(*repository.Application)
+	c.Equal("321", app.ID)
+
+	n = <-driver.Notification
+	c.Equal(ActionUpdate, n.Action)
```

Comment:
Thank you for adding unit tests. Could we use table-driven tests to be consistent with the rest of the tests in the repository please?